rholang/reference_doc/data_structures/README.md

```diff
@@ -1,13 +1,49 @@
 ## Data Structures
 
-### Structures
+All computation can be encoded in the raw rho calculus just as it can be in the lambda calculus. Convincing your self of this is an interesting and rewarding endeavor for the mathematically inclined. Rholang strives to be a useful general purpose programming language, and therefore introduces some additional native data structures.
 
-list, tuple, set, map
+All data structures are processes and [can be quote] todo insert link, to create channels.
```

Comment:
What does this mean? 
> [can be quote] todo insert link

Reply:
It should say "can be quoted". I'll fix that now. The todo is a note for me (or whomever) to link to the section of the manual about quoting once it is written.